### PR TITLE
use shared directory.build.props file

### PR DESCRIPTION
**Pull request by devhawk**

### CODE REVIEW

File: src/ApplicationLogs/ApplicationLogs.csproj [hunk 1]
```diff
@@ -1,9 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.0.0-preview3</Version>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <RootNamespace>Neo.Plugins</RootNamespace>
+    <PackageId>Neo.Plugins.ApplicationLogs</PackageId>
```
Comment (erikzhang) [inline src/ApplicationLogs/ApplicationLogs.csproj:4]: Do we need to explicitly add a PackageId for each project?

Comment (devhawk) [reply src/ApplicationLogs/ApplicationLogs.csproj:4]: Nope. Packages will default to the project name if PackageId is not explicitly specified. I figured the packages needed to use the nuget.org reserved ID prefix "Neo".

Comment (erikzhang) [reply src/ApplicationLogs/ApplicationLogs.csproj:4]: But not all of them are going to be published to the nuget.

Comment (devhawk) [reply src/ApplicationLogs/ApplicationLogs.csproj:4]: Neo-Express has dependencies on RpcClient + RpcServer already and will be taking a dependency on Oracle once it's merged. Those modules need to be published on nuget.org. Standardizing the naming for the remaining packages seems like a reasonable idea

File: src/LevelDBStore/LevelDBStore.csproj [hunk 2]
```diff
@@ -1,9 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.0.0-preview3</Version>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <RootNamespace>Neo</RootNamespace>
+    <PackageId>Neo.Plugins.LevelDBStore</PackageId>
```
Comment (erikzhang) [inline src/LevelDBStore/LevelDBStore.csproj:4]:
```suggestion
    <PackageId>Neo.Plugins.Storage.LevelDBStore</PackageId>
```

File: src/RocksDBStore/RocksDBStore.csproj [hunk 3]
```diff
@@ -1,8 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.0.0-preview3</Version>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <PackageId>Neo.Plugins.RocksDBStore</PackageId>
```
Comment (erikzhang) [inline src/RocksDBStore/RocksDBStore.csproj:4]:
```suggestion
    <PackageId>Neo.Plugins.Storage.RocksDBStore</PackageId>
```

File: src/RocksDBStore/RocksDBStore.csproj [hunk 4]
```diff
@@ -1,8 +1,7 @@
...
     <RootNamespace>Neo.Plugins.Storage</RootNamespace>
```
Comment (erikzhang) [inline src/RocksDBStore/RocksDBStore.csproj:5]:
```suggestion
```

File: src/RpcClient/RpcClient.csproj [hunk 5]
```diff
@@ -1,20 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <Version>3.0.0-preview3</Version>
-    <TargetFramework>netstandard2.1</TargetFramework>
-    <RootNamespace>Neo.Network.RPC</RootNamespace>
```
Comment (erikzhang) [inline src/RpcClient/RpcClient.csproj:6]: I think this line should be kept. `RpcClient` is not a plugin.

File: src/RpcClient/RpcClient.csproj [hunk 6]
```diff
@@ -1,20 +1,7 @@
...
-    <Authors>The Neo Project</Authors>
-    <PackageId>Neo.Network.RPC.RpcClient</PackageId>
-    <PackageTags>NEO;Blockchain</PackageTags>
-    <PackageProjectUrl>https://github.com/neo-project/neo-modules</PackageProjectUrl>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/neo-project/neo-modules.git</RepositoryUrl>
+    <PackageId>Neo.RpcClient</PackageId>
```
Comment (erikzhang) [inline src/RpcClient/RpcClient.csproj:4]:
```suggestion
    <PackageId>Neo.Network.RPC.RpcClient</PackageId>
```

File: src/RpcClient/RpcClient.csproj (hunk 5 above)
Comment (devhawk) [reply src/RpcClient/RpcClient.csproj:6]: That's why it doesn't have "Plugin" in the package name

File: src/RpcClient/RpcClient.csproj (hunk 6 above)
Comment (devhawk) [reply src/RpcClient/RpcClient.csproj:4]: Why do we need a longer package name?

File: src/RpcClient/RpcClient.csproj (hunk 5 above)
Comment (erikzhang) [reply src/RpcClient/RpcClient.csproj:6]: And the root namespace should also be different.

File: src/RpcClient/RpcClient.csproj (hunk 6 above)
Comment (erikzhang) [reply src/RpcClient/RpcClient.csproj:4]: Because it is consistent with its namespace.

**Review by shargon — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Review by shargon — APPROVED** [review]

**Pull request merged**